cfgov/core/tests/test_utils.py

```diff
@@ -90,3 +90,12 @@ def test_get_link_tags_does_not_match_aside(self):
             get_link_tags('outer <aside>inner <a href="">link</a></aside>'),
             ['<a href="">link</a>', ]
         )
+
+    def test_get_link_tags_spacing(self):
+        """ Handle a case where there's a conditional in a template, i.e.
+        <a {% if some_condition %}class="some-class"{% endif %}>, that renders
+        a space following the tag name, `<a >`"""
```

Comment:
A little docstring pedantry/clarity, if you're amenable:

```suggestion
        """Test for case where tag renders with only whitespace after tag name

        Template conditions within a tag, e.g.,
        `<a {% if some_condition %}class="some-class"{% endif %}>`
        can result in output with only whitespace following the tag name
        if the condition is false, like: `<a >`.
        """
```